examples/dispatch.rs

```diff
@@ -0,0 +1,32 @@
+/// Demonstrates usage of various dispatch calls
+///
+/// Usage: cargo run --example dispatch
+
+use hyprland::dispatch::*;
+
+fn do_call(desc: &str, action: DispatchType) -> hyprland::Result<()> {
+    println!("{}: {:?}", desc, action); 
+    Dispatch::call(action)?; 
+    std::thread::sleep(std::time::Duration::from_secs(2)); 
+    return Ok(());
+}
+
+fn do_toggle(desc: &str, action: DispatchType) -> hyprland::Result<()> { 
+    do_call(desc, action.clone())?; 
+    do_call(desc, action)?;
+    return Ok(());
+}
+
+fn main() -> hyprland::Result<()>{
+    do_call("Moving cursor to top left", DispatchType::MoveCursorToCorner(Corner::TopLeft))?;
+    do_call("Moving cursor to top right", DispatchType::MoveCursorToCorner(Corner::TopRight))?;
+    do_call("Moving cursor to bottom right", DispatchType::MoveCursorToCorner(Corner::BottomRight))?;
+    do_call("Moving cursor to bottom left", DispatchType::MoveCursorToCorner(Corner::BottomLeft))?;
+    do_call("Moving window to next workspace", DispatchType::MoveToWorkspace(WorkspaceIdentifierWithSpecial::Relative(1), None))?;
+    do_call("Moving window to previous workspace", DispatchType::MoveToWorkspace(WorkspaceIdentifierWithSpecial::Relative(-1), None))?;
+    do_toggle("Toggling fullsceen", DispatchType::ToggleFullscreen(FullscreenType::Maximize))?;
```

Comment:
also you should untoggle the fullscreen, also fix the typo

Reply:
Typo fixed. I'm dyslexic so it took a bit of hunting. The code already reverted the fullscreen toggle but this is probably clearer in the refactored code.